src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -28,12 +29,23 @@ public FindCommand(Set<String> keywords) {
     }
 
     /**
-     * Returns a copy of keywords in this command.
+     * Returns a copy of keywords, in small case, in this command.
      */
-    public Set<String> getKeywords() {
-        return new HashSet<>(keywords);
+    public Set<String> ToSmallCaseKeywords(Set<String> keywords) {
+        //Converting set to array to iterate through
+        String[] keywordsArray = keywords.toArray(new String[keywords.size()]);
+        for (int i = 0; i < keywordsArray.length; i++) {
+            keywordsArray[i] = keywordsArray[i].toLowerCase();
+        }
+        Set<String> keywordsInSmallCase = new HashSet<>(Arrays.asList(keywordsArray));
+        return keywordsInSmallCase;
```

Comment:
Good job implementing this additional method to convert the keywords to lower case. Remember to follow the naming convention for method though! (should be toSmallCaseKeywords instead)